plotmodel.py

```diff
@@ -1,6 +1,7 @@
 import copy, struct, threading, openmc
 import openmc.capi.plot as capi_plot
 from openmc.capi.plot import _PlotBase
+import openmc
```

Comment:
PEP8 note: you should separate imports into three sections:
1. standard library imports
2. third-party imports
3. local imports

each separated by a single line